doc/user/content/sql/create-source/webhook.md

```diff
@@ -0,0 +1,108 @@
+{{% /create-source/intro %}}
+
+
+## Syntax
+
+{{< diagram "create-source-webhook.svg" >}}
+
+## `webhook_body_format`
+
+{{< diagram "webhook-body-format.svg" >}}
+
+## `webhook_check_option`
+
+{{< diagram "webhook-check-option.svg" >}}
+
+Field                            | Use
+---------------------------------|--------------------------
+  _src_name_                     | The name for the source
+ **IN CLUSTER** _cluster_name_   | The [cluster](/sql/create-cluster) to maintain this source.
+ **INCLUDE HEADERS**             | Include a column named `'headers'` of type `map[text => text]` containing the headers of the request.
+ **CHECK**                       | Specify a boolean expression that is used to validate each request received by the source.
+
+### `BODY FORMAT` options
+
+Field    | Value    | Description
+---------|----------|-------------------------
+`TEXT`   | `text`   | Parses the body of a request as UTF-8 text. If the body is not valid UTF-8, a response of `400` Bad Request will be returned.
+`JSON`   | `jsonb`  | Parses the body of a request as JSON. If the body is not valid JSON, a respose of `400` Bad Request will be returned.
+`BYTES`  | `bytea`  | Does no parsing of the request, stores the body as it was received.
+
+### `CHECK WITH` options
+
+Field                  | Value               | Description
+-----------------------|---------------------|--------------
+`BODY`                 | `text` or `bytea`   | Provide a column `'body'` to the check expression. The column can be renamed with the optional **AS** _alias_ statement, and the type can be changed to `bytea` with the optional **BYTES** keyword.
+`HEADERS`              | `map[text=>text]` or `map[text=>bytea]` | Provide a column `'headers'` to the check expression. The column can be renamed with the optional **AS** _alias_ statement, and the type can be changed to `map[text => bytea]` with the optional **BYTES** keyword.
+`SECRET` _secret_name_ | `text` or `bytea`    | Provide a column _secret_name_ to the check expression, with the value of the [`SECRET`](/sql/create-secret) _secret_name_. The column can be renamed with the optional **AS** _alias_ statement, and the type can be changed to `bytea` with the optional **BYTES** keyword.
+
+## Webhook URL
+
+After creating a webhook source, requests can be sent to `https://<HOST>/api/webhook/<database>/<schema>/<src_name>`.
+Where `<HOST>` is the URL for your Materialize instance, which can be found on the [Materialize Web
+Console](https://console.materialize.com/). Then `<database>` and `<schema>` are the database and
+schema where you created your source, and `<src_name>` is the name you provided for your source at
+the time of creation.
+
+
+> **_Note:_** This is a public URL that is open to the internet and has no security. To validate that
+requests are legitimate see Validating Requests. For limits imposed on this endpoint see Request Limits.
```

Comment:
These notes are good uses for the callout shortcode. 
```suggestion
{{< callout >}}
> **_Note:_** This is a public URL that is open to the internet and has no security. To validate that
requests are legitimate see [Validating Requests](#validating-requests).
For limits imposed on this endpoint see [Request Limits](#request-limits).
{{< /callout >}}
```

Reply:
Ah ha! `callout` is what I was looking for, thanks!